librz/meson.build

```diff
@@ -197,9 +197,7 @@ bp_plugins = [
   'x86'
 ]
 
-core_plugins = [
-  'java'
-]
+core_plugins = []
```

Comment:
as said, I think this is wrong. IMO we should keep the java core plugin. We have plugins for everything, I don't see why `java` should be a "core" feature of rizin instead of a plugin.

In particular, you could still easiy use newshell without the need to have it outside the plugin architecture. I would register the `java` command inside the `RzCorePlugin.init` method. You would not be able to use the yaml thing anymore, but you can anyway register the java commands from C.

Reply:
reverted back